src/components/autocomplete/js/autocompleteController.js

```diff
@@ -208,6 +208,17 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   }
 
   /**
+   * Find the target for the wrap element which determines menu width
+   * @returns {*}
+   */
+  function getWrapTarget() {
+    for (var element = $element; element.length; element = element.parent()) {
```

Comment:
Okay, if it's relatively easy to do; why don't you go ahead with the `md-autocomplete-snap="width"` approach. That will give us the most flexibility.

If you can also add some docs to the Autocomplete about it's usage, that would be great!